samples/hapi-app/src/client/components/demo-cookies.jsx

```diff
@@ -60,7 +61,7 @@ class DemoCookies extends React.Component {
     return (
       <div>
         <h6 styleName={"custom.docs-header"}>Demo Non-Redux Universal Cookies</h6>
-        <p>Test cookie value: {cookieValue || "<NOT SET>"}</p>
+        <p styleName={"demo3.cookieTitle"}>Test cookie value: {cookieValue || "<NOT SET>"}</p>
```

Comment:
Was it your intent to make "demo3.cookieTitle" a string? If so, the braces are extraneous.

Reply:
good catch, that means we have to update all the code. :D